src/tickets/totp.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@nestjs/common';
+import { TOTP } from '@otplib/core';
+import { totp } from '@otplib/preset-default';
+
+@Injectable()
+export class TotpService extends TOTP {
```

Comment:
I don't know if we really need to create this as a service, we won't test it or mock it as it is already an external code. We can simply call the `otp` library directly.
What do you think @MonicaTanios @MichaelSafwatHanna?

Reply:
We figured we can override it to change the algorithm default options like the window for example to not make those changes inside the `TicketService`, maybe we can discuss a more simple solution.

Reply:
Agreed on removing it and calling the `otp` library directly.